frontend/src/components/customEditorInput.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { styleClasses } from '../views/projectEdit';
+import { FormattedMessage } from 'react-intl';
+import messages from './projectEdit/messages';
+
+export const CustomEditorTextInput = ({ name, value, handleChange }) => {
```

Comment:
As it doesn't have the possibility to be reused on other components, I prefer to let it on the `projectEdit/customEditorForm.js` file.